tldr/dfg_extractor.py

```
@@ -877,11 +917,44 @@ def _node_contains(self, parent, target) -> bool:
+            "package",
+            "import",
+            "type",
+            "struct",
+            "interface",
+            "fn",
+            "pub",
+            "mod",
+            "use",
+            "impl",
+            "trait",
+            "struct",
```

Ruff (0.14.13)

Sets should not contain duplicate item `"struct"`

Remove duplicate item

(B033)